support-frontend/package.json

```diff
@@ -42,7 +42,7 @@
       ".+\\.(js|jsx|ts|tsx)$": "./node_modules/babel-jest"
     },
     "transformIgnorePatterns": [
-      "/node_modules/(?!@guardian)",
+      "/node_modules/(?!@guardian|valibot)",
```

Comment:
https://stackoverflow.com/a/49676319
https://jestjs.io/docs/configuration#transformignorepatterns-arraystring
some sources that might help make sense of this, before adding we got 
`    Jest encountered an unexpected token

    This usually means that you are trying to import a file which Jest cannot parse, e.g. it's not plain JavaScript.

    By default, if Jest sees a Babel config, it will use that to transform your files, ignoring "node_modules".
`